src/tools/info/tl_info.c

```diff
@@ -471,6 +471,66 @@ static void print_md_info(uct_component_h component,
     ;
 }
 
+static void print_cm_attr(uct_worker_h worker, uct_component_h component,
```

Comment:
next time pls submit this as separate PR - it's doesn't really depend on adding sockcm